### ISSUE TITLE
Erro which appear randomly in kodi.log

### ISSUE THREAD

**Issue by Mannshoch** [rbuehlma/pvr.zattoo#74]
Found this three line of error which appear around all 12h 

```
2020-02-11 06:35:28.881 T:2711094128   ERROR: AddOnLog: Zattoo PVR Client: Parsing cache file [special://profile/addon_data/pvr.zattoo/cache/bfe2b09eaad7c598b35aa6be3f6b474d] failed.
2020-02-11 14:43:06.175 T:2812633968   ERROR: Could not find suitable input format: x-directory/normal
2020-02-11 18:36:09.893 T:2572907376   ERROR: AddOnLog: Zattoo PVR Client: Parsing cache file [special://profile/addon_data/pvr.zattoo/cache/7064c652aa432845deec7edd6a5fa48f] failed.
``

**Comment by rbuehlma** [comment]
Please delete the two mentioned files of the first and the third error. The second error is probably not related to this addon.

The files should be in the home folder of Kodi <home>/.kodi/userdata/addon_data/pvr.zattoo/cache/bfe2b09eaad7c598b35aa6be3f6b474d

**Comment by Mannshoch** [comment]
_May interesting, I use now the new Raspberry Pi 4 since some days._ 
This two files do not exist in that directory.

**Comment by rbuehlma** [comment]
In this case, they probably got already deleted by the cleanup process

**Comment by rbuehlma** [comment]
The error message from this Addon should be fixed with 18.1.14 (Leia) or 19.2.11 (Matrix)
